src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java

```diff
@@ -110,11 +113,12 @@ public Boolean addItem(Long bucketId, Long itemId) {
 
     @Override
     public Boolean deleteItem(Long bucketId, Long itemId) {
-        try (Statement statementBuckets = connection.createStatement()) {
-            String queryBucktes = String.format(
-                    "DELETE FROM buckets_items where bucket_id = %d and item_id = %d;",
-                    bucketId, itemId);
-            statementBuckets.executeUpdate(queryBucktes);
+        String queryBucktes =
+                "DELETE FROM buckets_items WHERE bucket_id = ? AND item_id = ?;";
+        try (PreparedStatement statementBuckets = connection.prepareStatement(queryBucktes)) {
+            statementBuckets.setLong(1, bucketId);
+            statementBuckets.setLong(2, itemId);
+            statementBuckets.executeUpdate();
         } catch (SQLException e) {
             logger.error("Can't delete item", e);
         }
```

Comment:
is it correct to return `true` everywhere?
maybe better return true in try block and return false in catch block?

Reply:
OK.

Reply:
the same in other methods:)

Reply:
OK:)